### ISSUE TITLE
GRPO微调Modelscope仓库下载的Telechat2 出现了报错

### ISSUE THREAD

**Issue by WjMessi1** [modelscope/ms-swift#3749]
**Describe the bug**
参考训练脚本链接：https://github.com/modelscope/ms-swift/blob/main/examples/train/grpo/lora_vllm.sh
Modelscope模型：TeleAI/TeleChat2-7B-32K
编写的训练启动代码：

MASTER_PORT=29501 CUDA_VISIBLE_DEVICES=0,1,2 NPROC_PER_NODE=2 swift rlhf --rlhf_type grpo --model ~/data/telechat_train/modelscope/TeleChat2-7B-32K --reward_funcs accuracy format  repetition --train_type lora --use_vllm true --vllm_device auto --vllm_gpu_memory_utilization 0.5 --vllm_max_model_len 8192 --lora_rank 8 --lora_alpha 32 --target_modules all-linear --torch_dtype bfloat16 --dataset 'AI-MO/NuminaMath-TIR#2000' --max_completion_length xunl 0.5 --dataset_num_proc 4 --num_generations 8 --temperature 0.9 --system '/root/data/telechat_train/modelscope/TeleChat2-7B-32K/prompt.txt' --log_completions true

有报错：

<img width="762" alt="Image" src="https://github.com/user-attachments/assets/727f0a83-3ab5-488f-b53e-d95bc4e565b1" />

<img width="755" alt="Image" src="https://github.com/user-attachments/assets/54033608-35e8-4001-afd3-c0f99da465e3" />

**Your hardware and system info**
Write your system info like CUDA version/system/GPU/torch version here(在这里给出硬件信息和系统信息，如CUDA版本，系统，GPU型号和torch版本等)

3卡80G A100 
torch 2.6.0
NVIDIA-SMI 550.144.03
Driver Version: 550.144.03
CUDA Version: 12.4
ms-swift: 3.2.2
modelscope: 1.24.1
vllm: 0.8.2

**Comment by hjh0119** [comment]
what does `--max_completion_length xunl 0.5` mean?

**Comment by WjMessi1** [comment]
> what does `--max_completion_length xunl 0.5` mean?

I'm sorry，the full directive is：

MASTER_PORT=29501 CUDA_VISIBLE_DEVICES=0,1,2 NPROC_PER_NODE=2 swift rlhf --rlhf_type grpo --model /root/data/telechat_train/modelscope/TeleChat2-7B-32K --reward_funcs accuracy format repetition --train_type lora --use_vllm true --vllm_device auto --vllm_gpu_memory_utilization 0.5 --vllm_max_model_len 8192 --lora_rank 8 --lora_alpha 32 --target_modules all-linear --torch_dtype bfloat16 --dataset 'AI-MO/NuminaMath-TIR#2000' --max_completion_length 1024 --num_train_epochs 1 --per_device_train_batch_size 16 --per_device_eval_batch_size 16 --learning_rate 1e-5 --gradient_accumulation_steps 1 --eval_steps 100 --save_steps 100 --save_total_limit 2 --logging_steps 5  --max_length 2048 --output_dir output --warmup_ratio 0.05 --dataloader_num_workers 4  --dataset_num_proc 4 --num_generations 8 --temperature 0.9 --deepspeed zero2 --system 'examples/train/grpo/prompt.txt' --log_completions true
